### PR TITLE
[Dashboard] Remove parseDescription

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR focuses on removing the `parseDescription` utility function from the `BatchTable` component and replacing its usage with a new JSX structure that wraps the description in a `ToolTipLabel` component.

### Detailed summary
- Deleted the import of `parseDescription` from `utils/parseDescription`.
- Replaced the `accessor` for the "Description" column in `BatchTable`:
  - Previously used `parseDescription(row.description)`.
  - Now uses a JSX structure with `ToolTipLabel` and a paragraph element to display `row.description`.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4826?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4826** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4826?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4825** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4825?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/core-ui/batch-upload/batch-table.tsx [hunk 1]
```diff
@@ -102,7 +102,11 @@ export const BatchTable: React.FC<BatchTableProps> = ({
       { Header: "Name", accessor: (row) => row.name },
       {
         Header: "Description",
-        accessor: (row) => parseDescription(row.description),
+        accessor: (row) => (
+          <ToolTipLabel label={row.description}>
+            <p className="whitespace-pre-wrap">{row.description}</p>
+          </ToolTipLabel>
```
Comment (MananTank) [inline apps/dashboard/src/core-ui/batch-upload/batch-table.tsx:108]: Add line-clamp-6

**Review by MananTank — CHANGES_REQUESTED** [review]
Add line-clamp-6

File: apps/dashboard/src/core-ui/batch-upload/batch-table.tsx (hunk 1 above)
Comment (kien-ngo) [reply apps/dashboard/src/core-ui/batch-upload/batch-table.tsx:108]: oops thanks

**Review by MananTank — APPROVED** [review]

**Pull request closed without merging**